### ISSUE TITLE
Blankstore crashes with 404

### ISSUE THREAD

**Issue by whirm** [mar-v-in/BlankStore#9]
This is with latest release
```
E/AndroidRuntime( 7829): FATAL EXCEPTION: AsyncTask #1
E/AndroidRuntime( 7829): Process: com.android.vending, PID: 7829
E/AndroidRuntime( 7829): java.lang.RuntimeException: An error occured while executing doInBackground()
E/AndroidRuntime( 7829): 	at android.os.AsyncTask$3.done(AsyncTask.java:304)
E/AndroidRuntime( 7829): 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
E/AndroidRuntime( 7829): 	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
E/AndroidRuntime( 7829): 	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
E/AndroidRuntime( 7829): 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
E/AndroidRuntime( 7829): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
E/AndroidRuntime( 7829): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
E/AndroidRuntime( 7829): 	at java.lang.Thread.run(Thread.java:818)
E/AndroidRuntime( 7829): Caused by: com.gc.android.market.api.Tools$HttpException: HTTP Code 404 : https://developers.google.com/accounts/docs/AuthForInstalledApps
E/AndroidRuntime( 7829): 
E/AndroidRuntime( 7829): 	at com.gc.android.market.api.Tools.postUrl(Tools.java:66)
E/AndroidRuntime( 7829): 	at com.gc.android.market.api.MarketSession.login(MarketSession.java:352)
E/AndroidRuntime( 7829): 	at com.gc.android.market.api.MarketSession.login(MarketSession.java:334)
E/AndroidRuntime( 7829): 	at com.android.vending.google.GooglePlayConnection.openConnectionSynced(GooglePlayConnection.java:131)
E/AndroidRuntime( 7829): 	at com.android.vending.google.DefaultGooglePlayConnection.startAppsRequestSynced(DefaultGooglePlayConnection.java:319)
E/AndroidRuntime( 7829): 	at com.android.vending.google.DefaultGooglePlayConnection.queryAppsByName(DefaultGooglePlayConnection.java:236)
E/AndroidRuntime( 7829): 	at com.android.vending.tasks.AppsTask.doInBackground(AppsTask.java:50)
E/AndroidRuntime( 7829): 	at com.android.vending.tasks.AppsTask.doInBackground(AppsTask.java:10)
E/AndroidRuntime( 7829): 	at android.os.AsyncTask$2.call(AsyncTask.java:292)
E/AndroidRuntime( 7829): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
E/AndroidRuntime( 7829): 	... 4 more
W/ActivityManager( 2012):   Force finishing activity 1 com.android.vending/.BlankActivity
```

Thanks for this amazing app!

**Comment by mar-v-in** [comment]
Are you on latest release? This looks exactly like #7 which was fixed in version 0.7.3

**Comment by whirm** [comment]
I checked again and it works now. I didn't up/downgrade since then. Maybe it was a server side fluke? (I am/was running 0.7.4)

I'll reopen it if I get that again.

Thanks!
